fastapi/encoders.py

```diff
@@ -45,10 +45,6 @@ def jsonable_encoder(
             "exclude_unset to keep in line with Pydantic v1, support for it will be "
             "removed soon."
         )
-    if include is not None and not isinstance(include, set):
-        include = set(include)
-    if exclude is not None and not isinstance(exclude, set):
-        exclude = set(exclude)
```

Comment:
Why did you remove these line?
Considering the type of variables, i think there is no possibility of removing these lines.

Reply:
When exclude is dict, that thing were wrong.
I user pydantic's advanced case.

Also see:
https://pydantic-docs.helpmanual.io/usage/exporting_models/#advanced-include-and-exclude